packages/rich-text/src/SdkProvider.tsx

```diff
@@ -8,7 +8,7 @@ interface SdkProviderProps {
 }
 
 function useSdk({ sdk }: SdkProviderProps) {
-  const sdkMemo = React.useMemo<FieldAppSDK>(() => sdk, [sdk.parameters.instance.activeLocales]); // eslint-disable-line -- TODO: explain this disable
```

Comment:
This line was necessary to rerender the rich text editor when the activeLocales (previously passed as instance params) would change. However now that we calculate them based on the `useActiveLocales` we don't seem to need this anymore

Reply:
I guess the new hook now deals with the rerenders instead